velox/type/Timestamp.cpp

```diff
@@ -234,94 +219,170 @@ bool Timestamp::epochToUtc(int64_t epoch, std::tm& tm) {
   return true;
 }
 
-std::string Timestamp::tmToString(
+StringView Timestamp::tmToStringView(
```

Comment:
I would suggest to write to pre-allocated `startPosition` and return its length, then the caller could wrap it as `string` or `StringView`

Reply:
Current impl. writes to pre-allocated memory. Maybe returning as StringView is better as it maintains the data pointer. Besides the length, the data position can also be important to avoid misuse.